galpy/potential/MovingObjectPotential.py

```diff
@@ -67,17 +49,34 @@ def __init__(self,orbit,amp=1.,GM=.06,
            2011-04-10 - Started - Bovy (NYU)
 
         """
-        Potential.__init__(self,amp=amp*GM,ro=ro,vo=vo,amp_units='mass')
-        if _APY_LOADED and isinstance(softening_length,units.Quantity):
-            softening_length= softening_length.to(units.kpc).value/self._ro
-        # Make sure we aren't getting physical outputs
+
+        Potential.__init__(self,ro=ro,vo=vo)
+        isList = isinstance(pot,list)
+        if isList:
+            pot=flatten(pot)
+            if isinstance(pot[0],Potential):
+                nonAxi = _isNonAxi(pot)
+                if nonAxi:
+                    raise NotImplementedError('MovingObjectPotential for non-axisymmetric potentials is not currently supported')
+                self._pot = copy.deepcopy(pot)
+            else:
+                raise RuntimeError("List input to 'MovingObjectPotential' must be of Potential-istances")
+        elif isinstance(pot,Potential):
+            if pot.isNonAxi:
+                raise NotImplementedError('MovingObjectPotential for non-axisymmetric potentials is not currently supported')
+            self._pot = copy.deepcopy(pot)
+        elif pot == None:
+            # Initialize a Plummer potential by default for comptatability & tests
+            pot = PlummerPotential(amp=1.0,b=1.0,ro=ro,vo=vo)
```

Comment:
This needs to have ``amp=GM``, ``b=softening_length`` so these get setup correctly. Not doing this is probably what makes the test in ``test_quantity`` fail on travis.